Lucidtech/Las/Client.cs

```diff
@@ -916,6 +933,90 @@ public object CreateDataset(string? name = null, string? description = null)
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>
+        /// Create a data bundle handle, calls the POST /models/{modelId}/dataBundles endpoint.
+        /// </summary>
+        /// <param name="modelId">Id of the model </param>
+        /// <param name="datasetIds">List of Dataset Ids that will be included in the data bundle
+        /// <param name="name">Name of the data bundle</param>
+        /// <param name="description">A brief description of the data bundle </param>
+        /// <returns>Data Bundle response from REST API</returns>
+        public object CreateDataBundle(
+            string modelId,
+            List<string> datasetIds,
+            string? name = null,
+            string? description = null
+        ) {
+            var body = new Dictionary<string, object> {
+                {"datasetIds", datasetIds},
+            };
+
+            if (name != null) {
+                body.Add("name", name);
+            }
+
+            if (description != null) {
+                body.Add("description", description);
+            }
+
+            RestRequest request = ClientRestRequest(Method.POST, $"/models/{modelId}/dataBundles", body);
+            return ExecuteRequestResilient(RestSharpClient, request);
+        }
+
+        /// <summary>List data bundles available, calls the GET /models/{modelId}/dataBundles endpoint.</summary>
+        /// <param name="modelId">Id of the model</param>
+        /// <param name="maxResults">Number of items to show on a single page</param>
+        /// <param name="nextToken">Token to retrieve the next page</param>
+        /// <returns>
+        /// JSON object with two keys:
+        /// - "dataBundles" which contains a list of data bundle objects
+        /// - "nextToken" allowing for retrieving the next portion of data
+        /// </returns>
+        public object ListDataBundles(string modelId, int? maxResults = null, string? nextToken = null) {
+            var queryParams = new Dictionary<string, object?>();
```

Comment:
Why is object optional here, does that make sense?

Reply:
Because it can be empty if no argument is provided